pep8.py

```diff
@@ -1056,17 +1057,61 @@ def is_binary_operator(token_type, text):
         if ('\n' in text or '\r' in text) and token_type != tokenize.STRING:
             line_break = True
         else:
-            if (is_binary_operator(token_type, text) and line_break and
+            if (_is_binary_operator(token_type, text) and line_break and
                     not unary_context and
-                    not is_binary_operator(previous_token_type,
-                                           previous_text)):
+                    not _is_binary_operator(previous_token_type,
+                                            previous_text)):
                 yield start, "W503 line break before binary operator"
             unary_context = text in '([{,;'
             line_break = False
             previous_token_type = token_type
             previous_text = text
 
 
+def break_after_binary_operator(logical_line, tokens):
```

Comment:
So, I haven't followed the `yield from` functionality all that closely, but if you don't mind, could you give an example of what this would look like with that feature? Getting a bit out of the scope of the PR, but might be something we come back to in the future.